cmd/admin/pause.go

```diff
@@ -0,0 +1,202 @@
+package main
+
+import (
+	"context"
+	"encoding/csv"
+	"errors"
+	"flag"
+	"fmt"
+	"io"
+	"os"
+	"strconv"
+	"strings"
+
+	"github.com/letsencrypt/boulder/identifier"
+	sapb "github.com/letsencrypt/boulder/sa/proto"
+)
+
+// subcommandPauseBatch encapsulates the "admin pause-batch" commands.
+type subcommandPauseBatch struct {
+	file string
+}
+
+var _ subcommand = (*subcommandPauseBatch)(nil)
+
+func (p *subcommandPauseBatch) Desc() string {
+	return "Batch pause a CSV containing (account, identifier type, list of identifier strings)"
+}
+
+func (p *subcommandPauseBatch) Flags(flag *flag.FlagSet) {
+	flag.StringVar(&p.file, "file", "", "Path to CSV file containing (account, identifier type, list of identifier strings)")
+}
+
+func (p *subcommandPauseBatch) Run(ctx context.Context, a *admin) error {
+	if p.file == "" {
+		return errors.New("the -file flag is required")
+	}
+
+	identifiers, err := a.readPausedAccountFile(p.file)
+	if err != nil {
+		return err
+	}
+
+	err = a.pauseIdentifiers(identifiers)
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+
+// subcommandUnpauseBatch encapsulates the "admin unpause-batch" commands.
+type subcommandUnpauseBatch struct {
+	file string
+}
+
+var _ subcommand = (*subcommandUnpauseBatch)(nil)
+
+func (u *subcommandUnpauseBatch) Desc() string {
+	return "Batch unpause a CSV containing (account, identifier type, list of identifier strings)"
+}
+
+func (u *subcommandUnpauseBatch) Flags(flag *flag.FlagSet) {
+	flag.StringVar(&u.file, "file", "", "Path to CSV file containing (account, identifier type, list of identifier strings)")
+}
+
+func (u *subcommandUnpauseBatch) Run(ctx context.Context, a *admin) error {
+	if u.file == "" {
+		return errors.New("the -file flag is required")
+	}
+
+	identifiers, err := a.readPausedAccountFile(u.file)
+	if err != nil {
+		return err
+	}
+
+	err = a.unpauseAccount(identifiers)
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+
+// csvData contains a golang representation of the data loaded in from a CSV
+// file for pausing and unpausing.
+type csvData struct {
+	accountID       int64
+	identifierType  identifier.IdentifierType
+	identifierValue []string
+}
+
+// pauseIdentifiers allows administratively pausing a set of domain names for an
+// account.
+func (a *admin) pauseIdentifiers(incoming []csvData) error {
```

Comment:
nit: move this to be next to subcommandPauseBatch.Run, the only method that calls it.